### DISCUSSION TITLE
Future Ready Talent Industry Sessions

### DISCUSSION THREAD

**Discussion by adityachaturvedii** [github/india#164] [Students]
I have completed the first video of industry sessions now for about 5 times yet it is still showing incomplete ?
Anyone having any idea what to do?

**Comment by Trinity-alpha** [comment]
Same problem mate.

**Comment by scmishra-cse** [comment]
I have also watched the whole first video of industry sessions Digitization and Automation in BFSI video without a single break or pause still didn't mark as complete.

**Reply by AyushSenDazzler** [reply]
**Give it a another try, I encountered same problem but now  i m able to complete 4 session.**

**Comment by kapil864** [comment]
Progress will be tracked on a timely basis, as what they told in orientation.
